pkg/scaffold/internal/templates/v2/main.go

```diff
@@ -64,7 +65,7 @@ func (*MainUpdater) GetPath() string {
 	return defaultMainPath
 }
 
-// GetPath implements Builder
+// GetIfExistsAction implements Builder
 func (*MainUpdater) GetIfExistsAction() file.IfExistsAction {
```

Comment:
Could we add further info here? WDYT?

Reply:
Same as https://github.com/kubernetes-sigs/kubebuilder/pull/1368#discussion_r383882586